libnmstate/netapplier.py

```diff
@@ -128,6 +125,34 @@ def _apply_ifaces_state(
     validator.validate_interfaces_state(desired_state, current_state)
     validator.validate_routes(desired_state, current_state)
 
+
+def _apply_edited_ifaces_without_checkpoint(
+    desired_state, verify_change, commit, rollback_timeout
+):
+    current_state = state.State(netinfo.show())
+
+    _prepare_desired_state_for_apply(current_state, desired_state)
+    new_interfaces = _list_new_interfaces(desired_state, current_state)
+
+    with _setup_providers():
+        state2edit = _create_editable_desired_state(
+            desired_state, current_state, new_interfaces
+        )
+        ifaces2edit, ifaces_edit_configs = _edit_interfaces(state2edit)
+        nm.applier.set_ifaces_admin_state(
+            ifaces2edit, con_profiles=ifaces_edit_configs
+        )
+        if verify_change:
```

Comment:
IMHO the state should be verified only once after all applies happened, so IMHO the `verify_change()` should not be part of this method

Reply:
Yes. You are right. Thanks!